Iteam1337/predictivemovement
# Feat: use formik

- Rebuilt the forms with formik
- Use custom validation for required fields 
- Normalize phone number (both checking if it is a correct swe phone number in the UI and normalize it in `engine-server`)


### How to test

- Create Booking 
- Edit Booking
- Create Transport 
- Edit transport 
- Do the whole flow to see if something breaks 😄
